jobs/function_test/prepare.sh

```diff
@@ -75,16 +75,8 @@ preparePackages() {
         popd
     done
 
-    cp -r build-deps/RackHD .
-    if [ -d "build-deps/on-build-config" ]; then
-        cp -r build-deps/on-build-config build-config
-    else
-        cp -r on-build-config build-config
-    fi
-    pushd build-config
-    ./build-config
-    popd
 
```

Comment:
@PengTian0 
Good point.
I'll and cp on-build-config here, on-build-config from manifest has high priority. If it exists it will overwrite the build-config from scm.